### ISSUE TITLE
Relative CSS File paths

### ISSUE THREAD

**Issue by prantikv** [yzane/vscode-markdown-pdf#5]
I have the markdown file and the css file in the same folder at the same level.

I added the file path to my css as following

```
{
     "markdown-pdf.breaks": true,
      "markdown-pdf.styles": [
        "E:/MarkDown/vscode/my.css"          // OK
    ]
}
```

This works but as the path is absolute moving the folder will break css

I also tried to add the following paths 
```
{
     "markdown-pdf.breaks": true,
      "markdown-pdf.styles": [
       ".//my.css"  //DID NOT WORK
       ".\my.css"  //DID NOT WORK
       "my.css"  //DID NOT WORK
    ]
}
```

I believe this should have worked but did not. Let me know if I am missing anything here.

**Comment by yzane** [comment]
@migrap Thank you so much.

**Comment by migrap** [comment]
@yzane you're welcome. Cheers.

**Comment by yzane** [comment]
Released 0.1.5 with this fix.
Please update!
